build.gradle

```diff
@@ -33,29 +33,43 @@ runIdeForUiTests {
     systemProperties['com.redhat.devtools.intellij.telemetry.mode'] = 'debug'
 }
 
+buildSearchableOptions {
+    if (System.getProperty("idea.platform.prefix") == null) {
+        systemProperties["idea.platform.prefix"] = ''
+    }
+}
+
 dependencies {
-    compile 'io.fabric8:knative-client:5.12.4'
-    compile 'com.redhat.devtools.intellij:intellij-common:1.8.0'
-    testCompile 'org.mockito:mockito-inline:3.8.0'
-    compile 'com.squareup.okio:okio:1.15.0'
+    implementation 'io.fabric8:knative-client:5.12.4'
+    implementation 'com.redhat.devtools.intellij:intellij-common:1.8.0'
+    testImplementation 'org.mockito:mockito-inline:4.6.1'
+    implementation 'com.squareup.okio:okio:3.1.0'
     // telemetry contributes annotations 13.0.0, so we need to declare newer version
-    compile 'org.jetbrains:annotations:16.0.1'
-    testCompile 'com.redhat.devtools.intellij:intellij-common:1.7.1:test'
+    implementation 'org.jetbrains:annotations:23.0.0'
+    testImplementation 'com.redhat.devtools.intellij:intellij-common:1.8.0:test'
 }
 
 configurations {
-    compile {
+    runtimeClasspath {
         exclude group: 'org.slf4j', module: 'slf4j-api'
     }
     integrationTestImplementation.extendsFrom testImplementation
     integrationTestRuntimeOnly.extendsFrom testRuntimeOnly
 }
 
 sourceSets {
+    main {
```

Comment:
Why do you need this ? It is the default by convention and other plugins don't require this